src/messages/pt.json

```diff
@@ -212,9 +213,9 @@
   "start_here_page_conversational.content.0.title": "VTEX Assisted Sales (SuiteShare)",
   "start_here_page_conversational.content.0.description": "Aprenda soluções para atrair mais clientes, organizar os atendimentos do seu time, capturar dados dos clientes, fechar vendas e mensurar resultados.",
   "start_here_page_conversational.content.0.link": "tracks/o-que-e-suiteshare",
-  "start_here_tutorials_page.title": "Tutoriais",
-  "start_here_tutorials_page.subtitle": "Guias com informações passo a passo para auxiliar você na operação da plataforma VTEX e no desenvolvimento do seu negócio.",
-  "start_here_tutorials_page.link": "Saiba mais",
```

Comment:
In the description, it says you meant to remove this string `start_here_tutorials_page.link`. But it is still in the EN and ES files and you seem to just have changed its key and value here in the PT file.

It is worth noting that the new string name is used in the tutorial page (`src/pages/docs/tutorials`), but it has no counterpart in the ES and EN files.
![image](https://github.com/user-attachments/assets/59438059-d661-4038-bc2b-52441d8894f7)

Reply:
I believe this is linked to the comment above about the `start_here_page.link` string.